### ISSUE TITLE
Where can I find reference materials for running (Jepsen-I/O + Elle) with database?

### ISSUE THREAD

**Issue by SungHoHong2** [jepsen-io/jepsen#480]
Hello Jepsen developers,

I'm currently working as a researcher in a company project that is inventing a new key-value store, and I have found this Jepsen I/O, which can be an excellent benchmark for checking the anomalies.

I'm checking the scope of the workload that needs to be done so that the Elle can be used to the company's new key-value store. For now, what I see from the provided test directory is that Elle can run the consistency model using the history hand-created by the developer.

**Are there any good examples that I can follow to run Elle with Jepsen I/O and a database as a whole? (not the unit-test)** I have read the amazing papers that Elle is actually applied in many different databases, and I want to see how the test case is done, so I can propose the Jepsen I/O as the main benchmark for checking the anomalies.

**Comment by aphyr** [comment]
Yes, you'll find links to the source code with examples of using Elle with Jepsen throughout Jepsen articles from the last year: http://jepsen.io/analyses. See, for instance, the tests for Redis, YugaByteDB, and Etcd.
